docs/configuration_and_usage.rst

```diff
@@ -0,0 +1,93 @@
+Configuration and usage
+=======================
+
+Getting around administrator interface
+--------------------------------------
+
+`Edx-enterprise` is integrated with edx-platform Django admin. It can be accessed at
+`$LMS_SERVER_ADDRESS/admin/enterprise` URL.
+
+.. image:: images/enterprise-admin.png
+
+So far, there are two items in the admin interface:
+
+* "Enterprise Customers" - lists and manages Enterprise Customer records.
+* "Enterprise Customer Users" - lists and manages enterprise customer students.
+
+.. image:: images/enterprise-customer.png
+
+Despite providing student management interface in Enterprise Customer Users section, `edx-enterprise` admin provides
+custom (and richer) interface for managing learners associated with particular Enterprise Customer in "Enterprise
+Customer" section, accessible via "Manage Learners" button in the top-right corner of Enterprise Customer edit
+interface.
+
+Manage Learners View
+--------------------
+
+Manage Learners view main goal is to provide means to:
+
+* Associate ("link") and disassociate ("unlink") existing students with Enterprise Customer.
+* Create and remove "future associations" (also known as "Pending Enterprise Customer Users").
+
+In addition, during linking student(s) to Enterprise Customer it is capable of enrolling them into Course or Program.
+
+.. image:: images/manage-learners.png
+
+Integrating with Course Catalog
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order for course enrollment features to work, `edx-enterprise` talks to `Course Catalog IDA`_. Instructions on
+setting up and configuring Course Catalog can be found at corresponding `docs section` of Course Catalog and in
+`edx-enterprise` `Pull Request #7`_ (describes developer setup, but useful for a broader audience).
+
+One particular quirk is that Course Catalog IDA have internal access management, so administrators should access
```

Comment:
@bradenmacdonald 

> do I need to create a separate account, or somehow register my existing LMS account?

As far as I can tell - yes you do. In two words, Catalog Service uses LMS as OAuth provider and only returns courses/programs a user can access. What exactly "can access" mean - better to ask @haikuginger, since he was implementing that; for testing we're using a superuser account created specifically for this task, both in Course Catalog and LMS. Really, see #7 testing instructions - that's the only source of knowledge about that I have.

@haikuginger you might want to update this part of the docs with more accurate info.